packages/app-js/src/snippets/storage-examples.ts

```diff
@@ -10,13 +10,16 @@ export const storageGetInfo: Snippet = {
   label: { color: 'blue', children: 'Storage', size: 'tiny' },
   code: `// Get chain state information
 // Make our basic chain state/storage queries, all in one go
-const [blockPeriod, validators, transferFee] = await Promise.all([
-  api.query.timestamp.blockPeriod(),
+
+const [minimumPeriod, validators, transferFee] = await Promise.all([
+  api.query.timestamp.minimumPeriod(),
   api.query.session.validators(),
   api.query.balances.transferFee()
 ]);
 
-console.log('blockPeriod in seconds: ' + blockPeriod.toNumber());
+const period = util.formatNumber(minimumPeriod.toNumber() * 2);
```

Comment:
(nit) Could we add a comment to explain the `* 2`, e.g it's related to aura implementation. This confused me at first :)

Reply:
Good point. My example is wrong either way... It should either be minimumPeriod in the console.log or block period ===  minimumPeriod*2 (with aura)